riotfile.py

```diff
@@ -2697,10 +2697,8 @@ def select_pys(min_version: str = MIN_PYTHON_VERSION, max_version: str = MAX_PYT
                 Venv(
                     pys=select_pys(min_version="3.9", max_version="3.11"),
                     command="pytest {cmdargs} tests/opentracer/test_tracer_tornado.py",
-                    # TODO: update opentracing tests to be compatible with Tornado v6.
-                    # https://github.com/opentracing/opentracing-python/issues/136
                     pkgs={
-                        "tornado": ["~=4.5.0", "~=5.1.0"],
```

Comment:
we can't change this yet until the removal, it is deprecated but should still work and be tested.